### PR TITLE
dev-libs/univalue: Bump to 1.0.5

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

File: dev-libs/univalue/univalue-1.0.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="C++ universal value object and JSON library"
+HOMEPAGE="https://github.com/jgarzik/univalue"
+SRC_URI="https://codeload.github.com/jgarzik/${PN}/tar.gz/v${PV} -> ${P}.tgz"
+
+LICENSE="MIT"
+SLOT="0/0"
```
Comment (juippis) [inline dev-libs/univalue/univalue-1.0.5.ebuild:13]: `SLOT="0"` is enough generally.

File: dev-libs/univalue/univalue-1.0.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+KEYWORDS="amd64 arm arm64 mips ppc ppc64 x86 amd64-linux x86-linux"
```
Comment (juippis) [inline dev-libs/univalue/univalue-1.0.5.ebuild:14]: You can't commit straight to stable. Repoman would've warned and stopped you here.

File: dev-libs/univalue/univalue-1.0.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
```
Comment (juippis) [inline dev-libs/univalue/univalue-1.0.5.ebuild:18]: These empty variables can be removed.

**Comment by luke-jr** [comment]
Changes made

**Pull request closed without merging**